### PR TITLE
SOV-2145: add Adjust - Borrow LOC maintenance states

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-2145

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx [hunk 1]
```diff
@@ -132,6 +135,11 @@ export const FormContent: FC<FormContentProps> = props => {
     [props.hasTrove],
   );
 
+  const isBorrowDisabled = useMemo(
+    () => borrowLocked && props.hasTrove && props.debtType === AmountType.Add,
```
Comment (creed-victor) [inline apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx:138]:
```suggestion
    () => borrowLocked && (props.hasTrove && props.debtType === AmountType.Add || !props.hasTrove),
```

File: apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ import { AmountType } from '../types';
 export type OpenTroveProps = {
   hasTrove: false;
   liquidationReserve: Decimal;
+  debtType: AmountType;
```
Comment (creed-victor) [inline apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx]: When opening trove, `hasTrove` is always `false` and always `true` for Adjusting.
So it means that you don't need to add `debtType` here, `hasTrove == false` is same as `debtType == AmountType.Add`

File: apps/frontend/src/app/3_organisms/ZeroLocForm/components/OpenCreditLine.tsx [hunk 3]
```diff
@@ -227,6 +227,7 @@ export const OpenCreditLine: FC<OpenCreditLineProps> = ({
       debtError={debtError}
       collateralError={collateralError}
       errors={errors}
+      debtType={AmountType.Add}
```
Comment (creed-victor) [inline apps/frontend/src/app/3_organisms/ZeroLocForm/components/OpenCreditLine.tsx]:
```suggestion
```

File: apps/frontend/src/app/3_organisms/ZeroLocForm/components/OpenCreditLine.tsx [hunk 4]
```diff
@@ -18,7 +18,7 @@ import {
   MIN_DEBT_SIZE,
 } from '../constants';
 import { useZeroData } from '../hooks/useZeroData';
-import { CreditLineSubmitValue } from '../types';
+import { AmountType, CreditLineSubmitValue } from '../types';
```
Comment (creed-victor) [inline apps/frontend/src/app/3_organisms/ZeroLocForm/components/OpenCreditLine.tsx]:
```suggestion
import { CreditLineSubmitValue } from '../types';
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

**Review by creed-victor — APPROVED** [review]

File: .changeset/six-seals-confess.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+"frontend": patch
+---
+
+SOV-2145: add Adjust - Borrow LOC
```
Comment (soulBit) [inline .changeset/six-seals-confess.md]:
```suggestion
SOV-2145: add Adjust - Borrow LOC maintenance states
```

**Review by soulBit — APPROVED** [review]

**Pull request merged**